tests/playbooks/e2e.yaml

```diff
@@ -20,16 +20,7 @@
         cdi_source:
           http:
             url: "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
-
-    - name: Wait for PVC to be ready
-      k8s_facts:
-        name: "{{ pvc_name }}"
-        kind: PersistentVolumeClaim
-        api_version: v1
-      register: pvc
-      until: "pvc.resources[0].status.phase == 'Bound' and 'cdi.kubevirt.io/storage.pod.phase' in pvc.resources[0].metadata.annotations and pvc.resources[0].metadata.annotations['cdi.kubevirt.io/storage.pod.phase'] == 'Succeeded'"
-      delay: 15
-      retries: 30
+        wait: yes
```

Comment:
Do we want to customize timeout?

Reply:
The default is quite high (5 minutes), as this is PVCs, so it's expected to take a while (especially with CDI involved). Tests passed, so I assume we're fine for now. If it takes more than 5 minutes, we likely have an issue with the CI.